src/space.js

```diff
@@ -40,11 +45,69 @@ class Space {
         await this._store._sync(numEntries)
         if (opts.onSyncDone) opts.onSyncDone()
       }
-      syncSpace()
+      this._syncSpacePromise = syncSpace()
       this.public = publicStoreReducer(this._store)
       this.private = privateStoreReducer(this._store, this._3id.getKeyringBySpaceName(nameToSpaceName(this._name)))
     }
   }
+
+  /**
+   * Join a thread. Use this to start receiving updates from, and to post in threads
+   *
+   * @param     {String}    name                    The name of the thread
+   * @param     {Object}    opts                    Optional parameters
+   * @param     {Boolean}   opts.noAutoSub          Disable auto subscription to the thread when posting to it (default false)
+   *
+   * @return    {Thread}                            An instance of the thread class for the joined thread
+   */
+  async joinThread (name, opts = {}) {
+    if (this._activeThreads[name]) return this._activeThreads[name]
+    const subscribeFn = opts.noAutoSub ? () => {} : this.subscribeThread.bind(this, name)
```

Comment:
why not subscribe here (or few lines below), whats the thought behind a user must post to a thread to subscribe?

Reply:
You might want to open threads and see the content without needing to subscribe and always look for updates in that thread.

Reply:
oh ok, yeah that makes sense, just the name seems to imply subscribing vs something like openThread